common/buildings/wc_temple.txt

```diff
@@ -0,0 +1,23 @@
+temple = {
+	# Cenarion Enclave
+	# hp_cenarion_enclave is hospital version
+	tp_cenarion_enclave = {
+		desc = hp_cenarion_enclave_desc
+		trigger = { always = no }
+		is_active_trigger = {
+			FROM = { can_rule_peacefully_trigger = yes }
+			location = { is_capital = yes }
+		}
```

Comment:
I don't think that this building should be visible in every holding

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/74903152-33b86880-53b9-11ea-89ae-8bb980cb20a3.png)
![image](https://user-images.githubusercontent.com/46576860/74903164-3c10a380-53b9-11ea-83e0-97849d82eb07.png)

</details>